internal/distro/rhel8/ami.go

```diff
@@ -359,14 +359,6 @@ func defaultEc2SapImageConfigX86_64(rd distribution) *distro.ImageConfig {
 	return SapImageConfig(rd).InheritFrom(defaultEc2ImageConfigX86_64(rd))
 }
 
-// common ec2 image build package set
-func ec2BuildPackageSet(t *imageType) rpmmd.PackageSet {
```

Comment:
Thanks for catching this.  I'll rebase.

Reply:
Turns out this is correct because the ec2 build root was used in Azure as well.
From current `main`: https://github.com/osbuild/osbuild-composer/blob/560756a16419aad4278103aed0950ca8a0236062/internal/distro/rhel8/azure.go#L462

Reply:
Fair enough, thanks for checking 😉